apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -215,6 +213,8 @@ export class ConstructFrameworkWorkflow {
       // TODO: fix the `JSONSchemaDto` type to enforce a non-primitive schema type.
       controlSchema: staticStep.template!.controls!.schema as JsonSchema,
       skip: (controlValues: Record<string, unknown>) => this.processSkipOption(controlValues, fullPayloadForRender),
+      disableOutputSanitization:
+        (staticStep.controlVariables?.disableOutputSanitization as boolean | undefined) ?? false,
```

Comment:
do we need to transfer this flat to boolean just in case we stored it in the db as "false"